src/config/config.ts

```diff
@@ -19,6 +19,8 @@ export const triggerSchedule = '13 13 * * 2';
 
 export const url = 'localhost:3001';
 
+export const maxFileLength = 2097152;
```

Comment:
There is good practice to hardcode long numbers as a multiplication result with the comment.
So 2 MB can be presented like: 2 * 1024 * 1024 // 2 MB
So it'd be easier to read.